WalletWasabi.Backend/Filters/IdempotentActionFilterAttributeImpl.cs

```diff
@@ -18,32 +19,36 @@ public IdempotentActionFilterAttributeImpl(IMemoryCache cache)
 			_cache = cache;
 		}
 
-		public override void OnActionExecuting(ActionExecutingContext context)
+		public override async Task OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
 		{
 			var request = context.HttpContext.Request;
 
-			var (cacheKey, body) = GetCacheEntryKey(request);
+			var (cacheKey, body) = await GetCacheEntryKeyAsync(request);
```

Comment:
ConfigureAwait(false)?

Reply:
These are server-side async tasks and in asp.net core at least there is no custom synchronization context, that's why we don't use it anywhere in the backend. Anyway, I think it is okay to use it in the backend also.